mrblib/mrb_open3.rb

```diff
@@ -1,4 +1,47 @@
 module Open3
+  # @param [Array<String>] - command to execute
+  # @return [String, String, Process::Status] - stdout, status
+  def capture2(*cmd)
+    stdout, _, status = capture3(*cmd)
```

Comment:
Why not just:
```suggestion
    stdout, stderr, status = capture3(*cmd)
    $stderr.print(stderr)
```

Reply:
Hi, thanks for the review.

Mostly inattention. I forgot to check back on what Ruby' `capture2` did with the error stream.